hackathon/templates/hackathon/hackathon_detail.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{% load static %}
+{% load my_tags %}
+{% block content %}
+<section class="h-100">
+    <h2 class="p-orange text-center">{{ hackathon.display_name }}</h2>
+    <article class="card shadow mb-3">
+        <div class="card-body">
+            <p class="card-text">{{ hackathon.description }}</p>
+            <h6 class="card-subtitle mb-2 text-muted"><strong>From</strong> {{ hackathon.start_date }} <strong>to</strong> {{ hackathon.end_date }}</h6>
+            {% if hackathon.organiser %}
+                <p>Organiser: <a href="#" class="p-blue card-link">{{ hackathon.organiser }}</a></p>
+            {% endif %}
+            {% if hackathon.end_date|event_ended %}
+                {% if request.user.is_staff %}
+                    {% include 'hackathon/includes/enrollstaff.html' %}
```

Comment:
It seems a bit much to create separate templates for 5 lines of code. I would just add the code directly in the template.

Reply:
I followed and respected he code of @ri.dearg for his original enroll.html. I do actually find it clearer with separate templates due to the form. But if you really want me to remove those enroll.html templates and add code directly in hackatondetail.html, I can do it

Reply:
@TravelTimN Hi Tim, could you have a look at this comment here and let me know if I should still apply the change requested by Stefan. I find it clearer with separate templates.